docs/adguard-vpn-for-windows/overview.md

```diff
@@ -97,22 +93,22 @@ All logs are stored locally on your device and you can send them to the support
 This is an experimental feature that enables AdGuard to use the advanced QUIC encryption protocol. It has many advantages, but the most notable one is that it can improve connection quality under non-ideal conditions, for example, when using mobile Internet or when connecting to public Wi-Fi networks. 
 
 
-### Split tunneling
+### App exclusions
 
-![Split tunnelling](https://cdn.adguard.com/content/kb/VPN/windows/split_tunneling_en.png)
+![Adding an app to exclusions](https://cdn.adguard.com/content/release_notes/vpn/windows/v2.0/add_app_en.png)
 
-AdGuard VPN encrypts the traffic not only of your browsers, but also of other apps installed on your computer. If you want to exclude certain applications from AdGuard's work, put them to the **Split tunneling** list. 
+AdGuard VPN not only encrypts the traffic of your browsers, but also of other apps installed on your computer. If you want to exclude certain applications from the tunnel, put them to the **App exclusions** list. 
```

Comment:
AdGuard VPN encrypts not only the traffic of your browsers, but also that of the applications installed on your device.
/
Not only does AdGuard VPN encrypt the traffic of your browsers, but also of other apps installed on your device.